client/src/pages/CoursePage.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Link } from "react-router-dom"; // Import Link for internal navigation
+
+const CoursePage = () => {
+  const courses = [
+    { id: 1, name: "Course 1", imgSrc: "https://picsum.photos/id/237/200/301" },
+    { id: 2, name: "Course 2", imgSrc: "https://picsum.photos/200/300" },
+    { id: 3, name: "Course 3", imgSrc: "https://picsum.photos/id/257/200/300" },
+    { id: 4, name: "Course 4", imgSrc: "https://picsum.photos/id/238/200/300" },
+    { id: 5, name: "Course 5", imgSrc: "https://picsum.photos/id/239/200/300" },
+    // Add more courses as needed
+  ];
+
+  const handleAddCourse = () => {
+    alert("Add Course button clicked!");
+  };
+
```

Comment:
Persistant users are not added yet, but you will eventually have to add logic for when you actually have a user or not.
If you don't have an user you redirect to student login, if you do have a user you just load the component normally